src/app/src/components/FilterSidebarFacilitiesTab.jsx

```diff
@@ -256,40 +259,54 @@ function FilterSidebarFacilitiesTab({
         </div>
     );
 
+    const componentSizes = [
+        64, // Navbar
+        107, // Sidebar heading
+        48, // Sidebar tabs
+        138, // Sidebar controls
+        51, // Footer
+        8, // Additional padding
+    ];
```

Comment:
Wonder if we should set these values as constants somewhere if they're hard-coded in multiple places?

Reply:
Good idea. Since we are styling with JS objects, we should have the luxury of just moving the values out to a shared constant.

Reply:
Turns out it is not that easy. There are `min-height` and `padding` directives coming in from our top level stylesheet as well as Material UI that are complicating things. I am going to instead add class tags to the elements that we need to measure and do the height calculation on the fly.

Reply:
Implemented a class-based height calculation in fixup abdafcf